core/chains/evm/types/models.go

```diff
@@ -165,6 +167,14 @@ func (h *Head) ChainHashes() []common.Hash {
 	return hashes
 }
 
+func (h *Head) LatestFinalizedHead() commontypes.Head[common.Hash] {
```

Comment:
Is this just for the unit-test? I don't see any other use

Reply:
It's not used. It was created for consumers. In particular I'm planning to use it in TXM's TX finalized state PR